[2023-01-04T15:04:30.864+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: titanic_etl_pipeline.create_dashboard_task manual__2023-01-04T14:59:13.547919+00:00 [queued]>
[2023-01-04T15:04:30.877+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: titanic_etl_pipeline.create_dashboard_task manual__2023-01-04T14:59:13.547919+00:00 [queued]>
[2023-01-04T15:04:30.878+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-01-04T15:04:30.880+0000] {taskinstance.py:1284} INFO - Starting attempt 2 of 2
[2023-01-04T15:04:30.880+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-01-04T15:04:30.913+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): create_dashboard_task> on 2023-01-04 14:59:13.547919+00:00
[2023-01-04T15:04:30.928+0000] {standard_task_runner.py:55} INFO - Started process 3376 to run task
[2023-01-04T15:04:30.931+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'titanic_etl_pipeline', 'create_dashboard_task', 'manual__2023-01-04T14:59:13.547919+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/titanic_etl.py', '--cfg-path', '/tmp/tmp_m9dm4be']
[2023-01-04T15:04:30.934+0000] {standard_task_runner.py:83} INFO - Job 63: Subtask create_dashboard_task
[2023-01-04T15:04:31.057+0000] {task_command.py:389} INFO - Running <TaskInstance: titanic_etl_pipeline.create_dashboard_task manual__2023-01-04T14:59:13.547919+00:00 [running]> on host 82367dbeb80f
[2023-01-04T15:04:31.157+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=titanic_etl_pipeline
AIRFLOW_CTX_TASK_ID=create_dashboard_task
AIRFLOW_CTX_EXECUTION_DATE=2023-01-04T14:59:13.547919+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-04T14:59:13.547919+00:00
[2023-01-04T15:04:31.201+0000] {logging_mixin.py:137} INFO - Dash is running on http://0.0.0.0:8050/
[2023-01-04T15:04:31.201+0000] {dash.py:1968} INFO - Dash is running on http://0.0.0.0:8050/

[2023-01-04T15:04:31.211+0000] {logging_mixin.py:137} INFO -  * Serving Flask app '***'
[2023-01-04T15:04:31.213+0000] {logging_mixin.py:137} INFO -  * Debug mode: off
[2023-01-04T15:04:31.214+0000] {logging_mixin.py:137} WARNING - Address already in use
[2023-01-04T15:04:31.215+0000] {logging_mixin.py:137} WARNING - Port 8050 is in use by another program. Either identify and stop that program, or start the server with a different port.
[2023-01-04T15:04:31.269+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2023-01-04T15:04:31.311+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
